docs/components/select.md

```diff
@@ -0,0 +1,25 @@
+# Select
+Select components are one of the most used elements in the web world. Here are the some examples of usage of the select component:
```

Comment:
"Select components are one of the most used elements in the web world." Bu kadar ovmeye gerek yok